tests/sentry/snuba/metrics/test_extraction.py

```diff
@@ -488,6 +489,10 @@ def test_spec_wildcard_escaping(query, expected_pattern) -> None:
         "value": [expected_pattern],
     }
 
+    # We also validate using Relay's glob implementation to make sure the escaping
+    # is interpreted correctly.
+    assert glob_match(title, expected_pattern, ignorecase=True)
```

Comment:
This function is not the same one used inside Relay for glob matching on rule conditions but it uses the same library at its core. For this reason, there is the chance that the two will diverge, making this test not 100% accurate but this is a good first step, also assuming that Relay won't change the globs impl in the near future.